flipperdevices/flipperzero-firmware
# [FL-3021] USB/BLE HID Remote icon fix

# What's new

- Removed BLE icon from `USB Remote` app
- BLE icon fix

# Verification 

- Build and upload external apps
- Check BLE state icon in `Bluetooth Remote` and `USB Remote` apps

# Checklist (For Reviewer)
